### PR TITLE
fix: extended deadline by 2 minutes

### PR DESCRIPTION
**Pull request by Bl20052005**

### CODE REVIEW

**Review by samderanova — COMMENTED** [review]
The deadline is also in the frontend, so please update that as well.

**Review by IanWearsHat — APPROVED** [review]
That is definitely an increase of 2 minutes

File: apps/site/src/app/(main)/apply/sections/ApplyConfirmation/ConfirmationDetails.tsx [hunk 1]
```diff
@@ -24,7 +24,7 @@ export default async function ConfirmationDetails({
 				and will be over the age of 18 by January 26th, 2024.
 			</p>
 			<strong className="text-lg text-[#FF2222]">
-				Applications are due on January 14th, 2024 at 11:59PM PST.
+				Applications are due on January 15th, 2024 at 12:01PM PST.
```
Comment (taesungh) [inline apps/site/src/app/(main)/apply/sections/ApplyConfirmation/ConfirmationDetails.tsx]: Chore [repeating]: Sorry the confusion, but we shouldn't publicly announce this change and leave it as 11:59.

Comment (Bl20052005) [reply apps/site/src/app/(main)/apply/sections/ApplyConfirmation/ConfirmationDetails.tsx]: alright then

Comment (Bl20052005) [reply apps/site/src/app/(main)/apply/sections/ApplyConfirmation/ConfirmationDetails.tsx]: is there anything else on the frontend that relies on dates that we should change, or is this purely a backend fix?

Comment (samderanova) [reply apps/site/src/app/(main)/apply/sections/ApplyConfirmation/ConfirmationDetails.tsx]: Sorry, I should have been clearer in my previous comment. We perform a datetime check in the frontend as well in the hasDeadlinePassed function, so please update the date object there too.

**Review by taesungh — APPROVED** [review]
Thanks for updating.

**Review by samderanova — APPROVED** [review]
Looks good, thanks for the quick fix! Please *squash* merge.

**Pull request merged**